src/web-api/controllers/stops.controller.js

```diff
@@ -21,4 +8,27 @@ function get(req, res) {
 //     .catch(e => next(e));
 // }
 
-export default { load, get };
+const STIB_API = process.env.STIB_API;
+
+function search(req, res) {
+  let by = req.query.by;
+  let term = req.query.term;
+
+  https.get(STIB_API + '/stops/proximity/1,1', function (respApi) {
```

Comment:
Keep in mind that will need the real coordinates (or search input : ID or Name) ; but good for now.

Reply:
If it closes #1 it's the search by id/name you should put here, not the search by proximity.